CHANGELOG.md

```diff
@@ -6,6 +6,35 @@ and this project uses `yyyy.rr[.pp]`, where `yyyy` is the year a patch is releas
 `rr` is a sequential release number (starting from `01`), and an optional two-digit
 sequential patch number (starting from `01`).
 
+## [2021.02] - 2021-05-19
```

Comment:
Should we begin using the three-field versioning for release notes?

Reply:
I had thought we were just going to include the third digit in the build systems for compatibility, and stick to the current naming scheme elsewhere. We can add it for the release notes too if you think that would make more sense, but it might be confusing since it's usually only added with patched versions.

Reply:
@bensonr I think @rem1776 is right.  It would be confusing to not match the tag and the release